mappings/net/minecraft/world/chunk/ChunkManager.mapping

```diff
@@ -6,6 +6,11 @@ CLASS net/minecraft/unmapped/C_bubysyxg net/minecraft/world/chunk/ChunkManager
 		ARG 3 create
 	METHOD m_gspauptw setMobSpawnOptions (Z)V
 		ARG 1 spawnMonsters
+	METHOD m_hwywidsl getChunk (IILnet/minecraft/unmapped/C_kogtzhzt;Z)Lnet/minecraft/unmapped/C_lwzmmmqr;
+		ARG 1 chunkX
```

Comment:
it would be nice if you could update the params for `isChunkLoaded` to make sure `x` and `z` have the prefix as well

Reply:
chunkX/Z vs x/z are actually used pretty interchangeably, I'll see if I can sort them out